### PR TITLE
Braze Currents Users Event field group updates

### PR DESCRIPTION
**Pull request by isturdy**
This PR includes the latest updates from the Braze Currents Users Events schemas.

### CODE REVIEW

File: extensions/braze/braze-currents-users-event.schema.json [hunk 1]
```diff
@@ -27,29 +27,70 @@
...
+                "P": "Prefer not to say"
+              }
+            },
+            "braze:phoneNumber": {
+              "title": "Phone Number",
+              "type": "string",
+              "description": "Phone number of the user in e164 format."
             }
-          },
-          "required": [
```
Comment (anandphatak) [inline extensions/braze/braze-currents-users-event.schema.json:50]: Just want to make sure you want to remove the `required` attribute. Once removed we won't be able to add it back in the future.

Comment (isturdy) [reply extensions/braze/braze-currents-users-event.schema.json:50]: We do--it isn't required in all events this schema covers, just in the first set we implemented.

File: extensions/braze/braze-currents-users-event.schema.json [hunk 2]
```diff
@@ -240,30 +445,143 @@
...
           }
         },
+        "braze:session": {
+          "title": "Session",
+          "type": "object",
+          "properties": {
+            "braze:duration": {
+              "title": "Duration",
+              "type": "number",
+              "description": "Duration of the session."
```
Comment (anandphatak) [inline extensions/braze/braze-currents-users-event.schema.json:563]: Is there an expected time unit? Then maybe it should be included in the description (like days/hours/minutes/seconds/milliseconds). You could also consider changing the field name to say `durationMilliseconds` or `durationSeconds` etc.

Comment (isturdy) [reply extensions/braze/braze-currents-users-event.schema.json:563]: That's a good idea. Changing the field name is complicated on our end, but I'll update the documentation once I have an answer.

**Review by anandphatak — APPROVED** [review]

**Pull request merged**